src/main/java/net/education/kyivstar/config/CreateSchema.java

```diff
@@ -0,0 +1,71 @@
+package net.education.kyivstar.config;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static net.education.kyivstar.services.util.ConfigSchemaPassUtils.DATABASE_PATH;
+import static net.education.kyivstar.services.util.ConfigSchemaPassUtils.TABLES_PATH;
+import static net.education.kyivstar.services.util.Utils.readSqlScriptFromFileStream;
+
+
+public class CreateSchema {
```

Comment:
Class names should be nouns, in mixed case with the first letter of each internal word capitalized. Try to keep your class names simple and descriptive. Use whole words-avoid acronyms and abbreviations (unless the abbreviation is much more widely used than the long form, such as URL or HTML). )(SchemaAndTableCreator)

Reply:
But you didn't rename it and use